solutions/usaco-738.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: usaco-738
+source: USACO
+title: Paired Up
+author: Sofia Yang
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_pairup_silver_open17.html)
+
+**Time Complexity:** $\mathcal{O}(N\log N)$
+
+
+<LanguageSection>
+
+<JavaSection>
+
+
+``` java
+import java.io.*;
+import java.util.*;
+
+public class pairup {
+	CodeSnip{Kattio}
+	
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio("pairup");
+		int N = io.nextInt();
+		ArrayList<pair> events = new ArrayList<>();
```

Comment:
i think we've decided to use interface declarations
so declare your type w/ `List` instead of `ArrayList`